storage/entities/account.js

```diff
@@ -404,7 +430,16 @@ class Account extends BaseEntity {
 	 * @return {*}
 	 */
 	update(filters, data, _options, tx) {
+		const atLeastOneRequired = true;
+
+		this.validateFilters(filters, atLeastOneRequired);
+
 		const objectData = _.omit(data, readOnlyFields);
+
+		if (_.isEmpty(objectData)) {
+			return Promise.resolve();
```

Comment:
Its intended to keep the existing behavior of the application. 

On many places application is giving data `objectData = {address: '1234L'}`, for which `address` can't be updated. So the `objectData` will get empty. If we throw error in that case, it will stop the promise chain to update accounts in the block processing. Then we have to change the business logic of the block processing to not stop the promise chain in this particular case.

I want to avoid any kind of change in business logic, to have confidence of compatibility with previous versions.